sdcm/utils/nemesis_utils/node_operations.py

```diff
@@ -0,0 +1,64 @@
+import contextlib
+import logging
+
+from typing import Optional
+from random import choice
+
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+@contextlib.contextmanager
+def use_network_firewall(target_node: "BaseNode"):
+    ports = [7001, 7000, 9042, 9142, 19042, 19142]
```

Comment:
Can we make it be configurable?
It seems useful for some other cases too where we may want to block only one/two ports...

Reply:
Added optional parameter, which could take list of ports to disable